Dockerfile

```diff
@@ -16,8 +16,7 @@ RUN BUILD_NUMBER=${BUILD_NUMBER} \
     BUILD_DATE=${BUILD_DATE} \
     BUILD_TIME=${BUILD_TIME} \
     BUILD_HASH=${BUILD_HASH} \
-    BUILD_VERSION=${BUILD_VERSION} \
-    make insgorund build
+    BUILD_VERSION=${BUILD_VERSION}
```

Comment:
actually, after deletion make command we don't need this argument. 
I suppose to you deleted extra `make build `stage

Reply:
changed it